Api/Controllers/ActivityController.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Application.Interfaces;
+using Application.Interfaces.Activity;
+using Domain.Entities.Activity;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Api.Controllers
+{
+    [AllowAnonymous]
+    public class ActivityController : BaseController
+    {
+        private readonly IActivityServices _activityServices;
+        private readonly IGenericRepository<ActivityEntity> _activityRepo;
+        public ActivityController(IActivityServices activityServices, IGenericRepository<ActivityEntity> activityRepo)
+        {
+            _activityServices = activityServices;
+            _activityRepo = activityRepo;
+        }
+        [HttpGet]
+        public async Task<IActionResult> GetActivitíes()
+        {
+            var activities = await _activityServices.List();
+            return Ok(activities);
+        }
+
+        [HttpGet("{id}")]
+        public string Get(int id)
```

Comment:
more specific name for this method. Don't use just Get, Post for controllers  methods